free-3.md

```diff
@@ -1,43 +1,52 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This project would serve users looking for a faster way to create and expand schedules. The DSL would "codify" scheduling by converting some text-based representation of the schedule into valid Google Calander or iCal format. This language might be a good way to meet the needs of people who enjoy the convenience of online schedules but find the process of filling it out and making changes or adding events very laborious. It might seem a bit silly at first to create an entire DSL for the sole purpose of saving a few minutes, but consider git. Github has both a terminal interface and a desktop alternative. Although some may consider the desktop interface to be limited in its functionality and less efficient than the terminal interface, others may find this more user-friendly and harder to mess up. Similarly, some may find terminal too hard to use or too easy to potentially mess up the repository. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This idea helps make scheduling easier by allowing for a text-based alternative. For instance, people who simply want to type up a few lines to generate a schedule for their semester classes rather than having to manually click and add each of their classes individually in the iCal interface. Rather than having to navigate through the calander to the specified month and date, potential users could simply write a one line command to schedule an event. Perhaps the biggest experience change with this DSL is that users would not have to actually open their calender app to input their schedule. As of now, users have to open their calender app and manually click through and create all their events.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because it streamlines the process for certain users. This DSL definitely appeals to a niche category of users that much prefer function over form. As someone always on the computer, it would be nice if I could simply schedule an event within a few seconds rather than having to open a browser, go to the calender app, and add it. 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I know that I am a forgetful person and would benefit exceptionally from a planner or calender of some sort; however, I find myself too lazy to actually go out of the way to create an event or reminder in Google Calenders or iCal, especially if the event is within a few days or even a few hours. It's ironic because it is these events that tend to be the most important. What excites me is the idea of simply oepning my computer, typing a command in terminal or some other environment and have the reminder automatically created.
 
```

Comment:
This is a slight extension of the original problem you mentioned. Now we're bringing reminders into the mix. This is fine but it definitely expands the scope of the problem. The DSL needs a way of reminding the user of an event, which could involve Google Calendar reminders or something completely different. It's also worth thinking about how this could be accessible across devices, since I usually appreciate reminders on my smartphone if I'm not going to be on my computer all day.